nixos/modules/services/web-servers/apache-httpd/default.nix

```diff
@@ -724,14 +728,15 @@ in
           ExecStart = "@${pkg}/bin/httpd httpd -f ${httpdConf}";
           ExecStop = "${pkg}/bin/httpd -f ${httpdConf} -k graceful-stop";
           ExecReload = "${pkg}/bin/httpd -f ${httpdConf} -k graceful";
-          User = "root";
+          User = cfg.user;
           Group = cfg.group;
           Type = "forking";
           PIDFile = "${runtimeDir}/httpd.pid";
           Restart = "always";
           RestartSec = "5s";
           RuntimeDirectory = "httpd httpd/runtime";
           RuntimeDirectoryMode = "0750";
+          AmbientCapabilities = [ "CAP_NET_BIND_SERVICE" ];
```

Comment:
With only 
```
AmbientCapabilities = [ "CAP_NET_BIND_SERVICE" ];
```
capabilities:
```
CapInh: 0000000000000400
CapPrm: 0000000000000400
CapEff: 0000000000000400
CapBnd: 0000003fffffffff
CapAmb: 0000000000000400
```
With
```
AmbientCapabilities = [ "CAP_NET_BIND_SERVICE" ];
CapabilityBoundingSet = [ "CAP_NET_BIND_SERVICE" ];
```
```
CapInh: 0000000000000400
CapPrm: 0000000000000400
CapEff: 0000000000000400
CapBnd: 0000000000000400
CapAmb: 0000000000000400
```
May add `CapabilityBoundingSet = [ "CAP_NET_BIND_SERVICE" ];` ?

Reply:
If the `service` runs as `wwwrun` then they should be no need for `CapabilityBoundingSet`. If the service runs a `root` (using `wwwrun` for children processes) it seems likely the sysadmin likely has needs we can't anticipate and using `CapabilityBoundingSet` might be overstepping. I really want to make it easy for a sysadmin to keep the current (and distro standard) behavior for `httpd` if they need to.

@flokli any thoughts on that?